interpreter/cling/lib/Utils/Signal.cpp

```diff
@@ -0,0 +1,21 @@
+//------------------------------------------------------------------------------
+// CLING - the C++ LLVM-based InterpreterG :)
```

Comment:
Why `InterpreterG`?

Reply:
An old bad joke to make `cling` an abbreviation ;-)

Reply:
Nevermind. I thought the author was Axel and that he asked me for a review.
Anyway, the `G` in the end is not a typo, it turns out :smiley: